File: before2025oct/baekjoon-c++/4153.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main()
{
    int array[3];

    while (1)
    {
        for (int i = 0; i < 3; i++)
        {
            cin >> array[i];
        }

        if (array[0] == 0 || array[1] == 0 || array[2] == 0)
        {
            break;
        }

        sort(array, array + 3);

        if (array[0] * array[0] + array[1] * array[1] == array[2] * array[2])
        {
            cout << "right" << endl;
        }
        else
        {
            cout << "wrong" << endl;
        }
    }

    return 0;
}

File: before2025oct/baekjoon-python/bronze/3/4153.py
while True:
    A = list(map(int, input().split()))
    A.sort()

    if A[0] == 0 and A[1] == 0 and A[2] == 0:
        break

    elif pow(A[0], 2) + pow(A[1], 2) == pow(A[2], 2):
        print("right")
    else:
        print("wrong")
